test/e2e/webdriver/chrome.js

```diff
@@ -3,21 +3,39 @@ const chrome = require('selenium-webdriver/chrome');
 const { ThenableWebDriver } = require('selenium-webdriver'); // eslint-disable-line no-unused-vars -- this is imported for JSDoc
 
 /**
- * Proxy host to use for HTTPS requests
+ * Default proxy host to use for HTTPS requests
  *
  * @type {string}
  */
-const HTTPS_PROXY_HOST = `${
-  process.env.SELENIUM_HTTPS_PROXY || '127.0.0.1:8000'
-}`;
+const DEFAULT_PROXY_HOST = '127.0.0.1:8000';
+
+/**
+ * Selenium Envar proxy host to use for HTTPS requests
+ *
+ * @type {string}
+ */
+const { SELENIUM_HTTPS_PROXY } = process.env;
+
+/**
+ * Determine the appropriate proxy server value to use
+ *
+ * @param {string|number} [proxyPort] - The proxy port to use
+ * @returns {string} The proxy server address
+ */
+function getProxyServer(proxyPort) {
```

Comment:
If the function is there to get the right value, the constants are just footguns to expose? So can be moved into the inner scope.

```suggestion
/**
 * Determine the appropriate proxy server value to use
 *
 * @param {string|number} [proxyPort] - The proxy port to use
 * @returns {string} The proxy server address
 */
function getProxyServer(proxyPort) {
  const DEFAULT_PROXY_HOST = '127.0.0.1:8000';
  const { SELENIUM_HTTPS_PROXY } = process.env;
```